### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by PicchiKevin** [docsifyjs/docsify#945]
Hi,

Here are some issues that I found regarding accessibility:
- Viewport not allowing to scale and zoom 
- The burger menu button must have discernible text
- The burger menu button must have a :focus styling
- Search plugin is text missing a label for the search input
- Search text input is also missing a :focus styling

Low contrast text according to [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/#contrast-minimum):
- Between #42b983 and #ffffff (ratio: 2.47)
- Between #858585 and #ffffff (ratio: 3.69)
- Between #e96900 and #ffffff (ratio: 3.04)
- Between #36bc98 and #ffffff (ratio: 2.38)

I also do like to note that some of the image alternate text can be improved.

I can submit a PR for the first 5 issues, but what should we do about the contrast ratio ?

>PS: Sorry about not following the templates, but this is not a bug report nor a feature request

**Comment by anikethsaha** [comment]
> The burger menu button must have discernible text

You meant a tooltip ?

> Search text input is also missing a :focus styling

what style are you planning to add in the focus?

Feel free to submit a PR

We can take a look at the ratio thing Future.

**Comment by PicchiKevin** [comment]
For the burher menu an aria label should do the trick although the button seems a bit small.

And for the text input here is what I was thinking of : 
<img width="312" alt="screenshot on the search bar of docsify with an outer glow" src="https://user-images.githubusercontent.com/4269168/69532477-cfc70500-0f6d-11ea-97f0-ec9eef28b16a.png">
(An outer glow should be enough I thing, with the theme color)

**Comment by anikethsaha** [comment]
> For the burger menu an aria label should do the trick although the button seems a bit small.

Can you try this and share the screenshot

And about the input, try something which will work with other theme too. And I think the outer glow is bit more, try reducing it little bit
